pkg/dist/buildmodule.go

```diff
@@ -56,10 +56,10 @@ type Target struct {
 	OS            string         `json:"os" toml:"os"`
 	Arch          string         `json:"arch" toml:"arch"`
 	ArchVariant   string         `json:"variant,omitempty" toml:"variant,omitempty"`
-	Distributions []Distribution `json:"distributions,omitempty" toml:"distributions,omitempty"`
+	Distributions []Distribution `json:"distros,omitempty" toml:"distros,omitempty"`
```

Comment:
Not sure how many people started using this new fields, but I will need to leave some **Note** in the release note for pack 0.34.0

Reply:
The `paketo` team (and Broadcom commercial teams) ran into an issue around this. Specifically, newer versions of the lifecycle require `[[targets.distributions]]` in `buildpack.toml` but `pack build` does not pick up those fields correctly.